src/modules/link.cpp

```diff
@@ -1,22 +1,123 @@
 //
 // Link Module
 //
-
+#include <Arduino.h>
+#include <FastCRC.h>
+#include "../pt/pt.h"
 #include "../modules/link.h"
 #include "../modules/module.h"
+#include "../hal/serial.h"
+
+FastCRC16 CRC16;  // this is the class to create crc16 to match the CCITT crc16 that rpi is using
 
-// Public Variables
-// TODO: Fix this variable name :(
+// Private Variables
 struct link comm;
+static struct pt serialThread;
```

Comment:
Please separate the private (Static) and public variables as the other modules do

Reply:
variables in serial.cpp and link.cpp have been separated like other modules.